### PR TITLE
US-3.3.5: Fix total points issues

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/profileStats/components/TotalPoints.vue [hunk 1]
```diff
@@ -101,6 +101,10 @@
         getGamificationPointsStats().then(
           (data) => {
             this.option.series[0].data = JSON.parse(JSON.stringify(data).split('"label":').join('"name":'));
+            for(let i=0;i<data.length;i++) {
+              const string = this.option.series[0].data[i].name;
+              this.option.series[0].data[i].name = string.charAt(0).toUpperCase() + string.slice(1);
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/profileStats/components/TotalPoints.vue]: this.option.series[0].data[i].name = this.option.series[0].data[i].name.toUpperCase() not working ?

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/profileStats/components/TotalPoints.vue]: that make all the letters in capital letters

Comment (azayati) [reply digital-workplace-webapps/src/main/webapp/vue-app/profileStats/components/TotalPoints.vue]: ok juste put optionSeriesName instead of string

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/profileStats/components/TotalPoints.vue]: Done

**Pull request merged**